libs/common-services/src/test/java/uk/gov/di/ipv/core/library/service/ConfigServiceTest.java

```diff
@@ -405,15 +370,29 @@ void shouldReturnListOfClientRedirectUrls(String testDataSet, String featureSet)
     }
 
     @ParameterizedTest
-    @CsvSource({"FEATURE_FLAGS,", "FEATURE_FLAGS,FS07"})
+    @CsvSource({"FEATURE_FLAGS,", "FEATURE_FLAGS,'FS07,DS01'", "FEATURE_FLAGS,'DS01,FS07'"})
     void shouldGetNamedFeatureFlag(String testDataSet, String featureSet) {
         environmentVariables.set("ENVIRONMENT", "test");
-        configService.setFeatureSet(featureSet);
+        List<String> fsList =
+                featureSet != null ? List.of(featureSet.split(",")) : Collections.emptyList();
+        configService.setFeatureSet(fsList);
         TestConfiguration testConfiguration = TestConfiguration.valueOf(testDataSet);
         testConfiguration.setupMockConfig(ssmProvider);
-        assertEquals(
-                Boolean.parseBoolean(testConfiguration.getExpectedValue(featureSet)),
-                configService.enabled(TestFeatureFlag.TEST_FEATURE));
+        if (featureSet == null) {
+            assertEquals(
+                    Boolean.parseBoolean(testConfiguration.getExpectedValue(featureSet)),
+                    configService.enabled(TestFeatureFlag.TEST_FEATURE));
+        } else {
+            if (fsList.get(0).equals("FS07")) {
+                assertEquals(
+                        Boolean.parseBoolean(testConfiguration.getExpectedValue(fsList.get(0))),
+                        configService.enabled(TestFeatureFlag.TEST_FEATURE));
+            } else {
+                assertEquals(
+                        Boolean.parseBoolean(testConfiguration.getExpectedValue(fsList.get(1))),
+                        configService.enabled(TestFeatureFlag.TEST_FEATURE));
```

Comment:
I think it's pretty unclear having conditional assertions in the test - perhaps this should just be 3 different tests (with no parameterisation)? That way you can write the assertions you want, and give them meaningful names.

If you want to share the setup, you could pull out a method, but tbh I'm not that fussed.

Reply:
Done - refactor test methods,  created different test methods.